frontend/src/pages/dashboard/components/sidebar/sidebar.tsx

```diff
@@ -34,11 +35,9 @@ const Sidebar: React.FC<Properties> = ({
           <li key={tab.name}>
             <Button
               label={isCollapsed ? '' : tab.name}
-              className={[
-                'h5',
-                styles.btn,
-                { [styles.active]: selectedTab === tab.name },
-              ]}
+              className={getValidClassNames('h5', styles.btn, {
+                [styles.active]: checkActiveTab(location.pathname, tab.name),
```

Comment:
const isActiveTab = checkIsActiveTab

className={getValidClassNames('h5', styles.btn, isActiveTab && styles.active)}

Reply:
I can't do that. Before that, we mapping TABS.
`      {TABS.map((tab) => (
          <li key={tab.name}>
            <Button
              label={isCollapsed ? '' : tab.name}
              className={getValidClassNames('h5', styles.btn, {
                [styles.active]: checkActiveTab(location.pathname, tab.name),
              })}
              frontIcon={tab.icon}
              variant="text"
              onClick={handleTabClick(tab.name)}
            >`